sw/device/silicon_creator/rom/doc/bootstrap.md

```diff
@@ -0,0 +1,193 @@
+# Bootstrap Protocol
+
+## Objective
+
+The goal of this proposal is to describe a bootstrap protocol for the OpenTitan chip which can be  implemented in the OpenTitan mask ROM while providing a standards-based interface which permits Silicon Owners the ability to bootstrap the chip without specialized equipment.
```

Comment:
Is this still a proposal, or what we've implemented?

Reply:
This is now a description of what is implemented.  I've removed the "Objective" sectin.